control_msgs/action/AntipodalGripperCommand.action

```diff
@@ -0,0 +1,11 @@
+sensor_msgs/JointState command # The velocity and effort fields are optionally and not used if empty.
```

Comment:
Suggest adding comments to the top of this file explaining what `AntipodalGrippers` are and a high level description of the expected behavior when calling this interface.

```suggestion
sensor_msgs/JointState command # The velocity and effort fields are option and not used if empty.
```

Reply:
Done